examples/managed-identity/kubernetes-msi.json

```diff
@@ -42,8 +43,8 @@
       }
     },
     "servicePrincipalProfile": {
-      "servicePrincipalClientID": "",
-      "servicePrincipalClientSecret": ""
+      "servicePrincipalClientID": "msi",

```

Comment:
I like the flow Anubhuti purposes and in general want to slim down cluster configs wherever we can...